main.js

```diff
@@ -160,4 +160,124 @@ const inventory = [
     originalStock: 10,
     sold: 8,
   },
-];
\ No newline at end of file
+];
+
+//Opdracht 1a: Gebruik een array-methode om een array te maken met alle tv-type namen. Log de uitkomst in de console.
+const nameOfTvType = inventory.map((tvType) => {
+  return 'tv-type: ' + tvType.type;
+});
+console.log(nameOfTvType);
+
+//Opdracht 1b: Gebruik een array-methode om alle tv's te verzamelen (de hele objecten) die volledig uitverkocht zijn. Log de uitkomst in de console.
+const tvOfSold = inventory.filter((soldTv) => {
+  return soldTv.sold > 0;
+    });
+console.log(tvOfSold);
+
+//Opdracht 1c: Gebruik een array-methode om alle tv's te verzamelen (de hele objecten) die over AmbiLight beschikken. Log de uitkomst in de console.
+const tvOfAmbiLight = inventory.filter((ambiLight) => {
+  return ambiLight.options.ambiLight === true;
+});
```

Comment:
Goed gedaan! Je zou dit nog korter kunnen schrijven:
```suggestion
const tvOfAmbiLight = inventory.filter(ambiLight => ambiLight.options.ambiLight);
```